ext/SymbolicsGroebnerExt.jl

```diff
@@ -318,14 +318,15 @@ function Symbolics.solve_multivar(eqs::Vector, vars::Vector{Num}; dropmultiplici
 end
 
 # Helps with precompilation time
-PrecompileTools.@setup_workload begin
-    @variables a b c x y z
-    simple_linear_equations = [x - y, y + 2z]
-    equations_intersect_sphere_line = [x^2 + y^2 + z^2 - 9, x - 2y + 3, y - z]
-    PrecompileTools.@compile_workload begin
-        symbolic_solve(simple_linear_equations, [x, y], warns=false)
-        symbolic_solve(equations_intersect_sphere_line, [x, y, z], warns=false)
-    end
-end
+# PrecompileTools.@setup_workload begin
+#     @variables a b c x y z
+#     simple_linear_equations = [x - y, y + 2z]
+#     equations_intersect_sphere_line = [x^2 + y^2 + z^2 - 9, x - 2y + 3, y - z]
+#     PrecompileTools.@compile_workload begin
+#         symbolic_solve(simple_linear_equations, [x, y], warns=false)
+#         symbolic_solve(equations_intersect_sphere_line, [x, y, z], warns=false)
+#     end
+# end
+# Disabled due to https://github.com/JuliaLang/julia/pull/55589
```

Comment:
this should just be removed then?

Reply:
just keep it separate?

Reply:
It can be re-added once Julia gets the ability for extensions to depend on other extensions

Reply:
it won't happen.

Reply:
I should write a better doc for it. Will copy the one in #1310

Reply:
Sure, I'll remove it then